Packs/Humio/Integrations/Humio/README.md

```diff
@@ -0,0 +1,811 @@
+>| WebHookNotifier | BTkuj8QArhIFMh_L39FoN0tnyTUEXplc | Null Webhook | bodyTemplate: {<br/>  "repository": "{repo_name}",<br/>  "timestamp": "{alert_triggered_timestamp}",<br/>  "alert": {<br/>    "name": "{alert_name}",<br/>    "description": "{alert_description}",<br/>    "query": {<br/>      "queryString": "{query_string} ",<br/>      "end": "{query_time_end}",<br/>      "start": "{query_time_start}"<br/>    },<br/>    "notifierID": "{alert_notifier_id}",<br/>    "id": "{alert_id}",<br/>    "linkURL": "{url}"<br/>  },<br/>  "warnings": "{warnings}",<br/>  "events": {events},<br/>  "numberOfEvents": {event_count}<br/>}<br/>headers: {"Content-Type": "application/json"}<br/>ignoreSSL: false<br/>method: POST<br/>url: http://localhost |
+>| WebHookNotifier | AQs6CuWm-uyXfYaNzwMyDGTX4S4qyAez | other | bodyTemplate: BODY<br/>headers: {"Content-Type": "application/json"}<br/>ignoreSSL: false<br/>method: POST<br/>url: http://localhost |
+
+
+### humio-delete-alert
+***
+Delete alert in Humio
+
+
+#### Base Command
+
+`humio-delete-alert`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| repository | Repository to use | Required | 
+| id | ID of the alert to be deleted | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Humio.Humio-delete-alert | Unknown | Details of the deletion | 
+
+
+#### Command Example
+```!humio-delete-alert repository=sandbox id=dIn3uuIvY4Gz90Bt2Dn2mVtDuB11ZUl2```
+
+#### Context Example
+```
+{}
+```
+
+#### Human Readable Output
+
+>Command executed. Status code <Response [204]>
+
+### humio-get-notifier-by-id
+***
+Get notifier from Humio by id
+
+
+#### Base Command
+
+`humio-get-notifier-by-id`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| repository | Repository to use | Required | 
+| id | ID to use | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Humio.Notifier | Unknown | Details of the notifier | 
+
+
+#### Command Example
+```!humio-get-notifier-by-id repository=sandbox id=BTkuj8QArhIFMh_L39FoN0tnyTUEXplc```
+
+#### Context Example
+```
+{
+    "Humio": {
+        "Notifier": {
+            "entity": "WebHookNotifier",
+            "id": "BTkuj8QArhIFMh_L39FoN0tnyTUEXplc",
+            "name": "Null Webhook",
+            "properties": {
+                "bodyTemplate": "{\n  \"repository\": \"{repo_name}\",\n  \"timestamp\": \"{alert_triggered_timestamp}\",\n  \"alert\": {\n    \"name\": \"{alert_name}\",\n    \"description\": \"{alert_description}\",\n    \"query\": {\n      \"queryString\": \"{query_string} \",\n      \"end\": \"{query_time_end}\",\n      \"start\": \"{query_time_start}\"\n    },\n    \"notifierID\": \"{alert_notifier_id}\",\n    \"id\": \"{alert_id}\",\n    \"linkURL\": \"{url}\"\n  },\n  \"warnings\": \"{warnings}\",\n  \"events\": {events},\n  \"numberOfEvents\": {event_count}\n}",
```

Comment:
I can suggest to better format the output/human readable in this case. Print only what is necessary and important